docs/contributing/3.0-development/index.rst

```diff
@@ -24,4 +24,15 @@ please file an issue in Redmine (related to https://pulp.plan.io/issues/2247) to
 track the conversion of the dependency to support Python 3 and begin working with
 upstream to convert the package.
 
+Docstrings
+----------
 
+PUP-2  TODO(asmacdo)link adopted the use of Google Style Docstrings TODO(asmacdo)link. When developers port code from Pulp 2 to Pulp 3, they should also convert all the docstrings to the new style.
```

Comment:
I went to review this, but it looks like it is still a work in progress. We shouldn't commit TODOs to the docs so maybe put a WIP? I can review whenever if you put a comment w/ my nick on it.